docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,404 @@
+---
+title: Server Access Getting Started Guide | Teleport Docs
+description: Getting started with Teleport Server Access.
+---
+
+# Getting Started
+
+Teleport Server Access provides full SSH capabilities out-of-the-box:
+
+1. Teleport's `tsh` combines support for SSH and SCP in one convenient local command-line tool.
+2. Each Teleport Node can be configured into SSH mode (Teleport Node) and run as an enhanced SSH server.
+
+The steps below involve:
+
+1. Creating a Teleport Auth Node cluster we'll add a Teleport SSH Node to.
+2. SSH into the cluster without external SSH dependencies (using `tsh` as an SSH client).
+3. Introspecting the cluster using advanced `tsh` features.
+
+## Prerequisites
+
+- A Linux environment (such as Ubuntu 20.05, CentOS 8.0-1905, or Debian 10) within which we'll create multiple instances.
+- A registered Doman Name.
+- A Two-Factor Authentication app (such as Authy or Google Authenticator).
+- Teleport (=teleport.version=) installed locally (for `tsh`).
+
+<Admonition type="warning" title="Warning">
+  The examples below may include the use of the `sudo` keyword to make following each step easier when creating resources from scratch. However, we generally discourage the use of `sudo` in **real-world production environments** per the *Principle of Least Privilege* (POLP). 
+</Admonition>
+
+## Step 1/5. Create a cluster
+
+<Admonition type="note" title="Note">
+  This guide involves configuring your Teleport Nodes into the *Bastion* pattern so that [only a single Node can be accessed publically](https://goteleport.com/blog/ssh-bastion-host/).
+</Admonition>
+
+<Admonition type="tip" title="Tip">
+  Multiple Teleport Nodes can be run on the same Linux instance. This Guide details how to launch and connect two Nodes on different devices.
+</Admonition>
+
+1. Verify you have a registered domain name. 
+
+  - We'll create one subdomain `tele.example.com` that will be publicly available.
+
+2. Create two new instances of your desired Linux distribution (such as Ubuntu 20.05, CentOS 8.0-1905, or Debian 10).
+
+   i. The first instance will host a public-facing Auth Node `tele.example.com` that will serve as the *Bastion Host* (following the recommended *Bastion*-pattern). As such, leave the following ports open:
+
+   | Port | Service | Description |
+   | - | - | - |
+   | 3023 | Proxy | SSH port clients connect to. A proxy will forward this connection to port `#3022` on the destination Node. |
+   | 3025 | Auth | SSH port used by the Auth Service to serve its API to other Nodes in a cluster. |
+   | 3080 | Proxy | HTTPS connection to authenticate `tsh` users and web users into the cluster. The same connection is used to serve a Web UI. |
```

Comment:
Ah, my fault - I did test this and was unable to connect to `3080` through my web client. Hence, the addition of port `443`. I will reverify since I probably missed something.

Also, elsewhere in the docs and blog we refer people to the Home - Getting Started doc for implementing the Bastion pattern:

1. https://goteleport.com/docs/getting-started/ <- lists  `3023`, `3024`, `3025`, and `443` as being required.
2. https://goteleport.com/blog/ssh-bastion-host/ <- directs people to use the above

Should that be updated too or would you like to keep that and direct the existing blog content to this new article?

Reply:
I think port `443` is required to add an initial user since they must accept the invite. And, this is required to tsh into the cluster later on. Is my understanding correct?

Reply:
It's either 443 or 3080 depending on the configuration. These ports are interchangeable. Let's use 443 then to be consistent.

Reply:
Great thanks - I'll drop the `3080` and `3025` options after a bit more tinkering for the minimal scenario here.

I see in the YAML we generate:
```
listen_addr: 0.0.0.0:3025
  cluster_name: bastion.example.com
  ....
  enabled: "yes"
  listen_addr: 0.0.0.0:3023
  web_listen_addr: :443
  public_addr: bastion.example.com:443
```